install/build.py

```diff
@@ -164,13 +164,21 @@ def get_compiler_setting(use_hip):
         else:
             define_macros.append(('CUPY_NO_NVTX', '1'))
 
-    cub_path = os.environ.get('CUB_PATH', '')
-    if os.path.exists(cub_path):
-        # for <cupy/complex.cuh>
-        cupy_header = os.path.join(os.path.dirname(os.path.realpath(__file__)),
-                                   '../cupy/core/include')
-        include_dirs.append(cupy_header)
-        include_dirs.append(cub_path)
+    # for <cupy/complex.cuh>
+    cupy_header = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                               '../cupy/core/include')
+    cub_path = None
+    if cuda_path:
+        cub_path = os.path.join(cuda_path, 'include', 'cub')
```

Comment:
Just wanna confirm: this line means `util_namespace.cuh` will live under `/usr/local/cuda/cub/cub/` instead of `/usr/local/cuda/cub/`. Is this correct?

Reply:
Yes, that is correct.
```
    [ CUDA 11.0 ]
    $ ls /usr/local/cuda/include/cub/util_namespace.cuh 
    /usr/local/cuda/include/cub/util_namespace.cuh
```

Reply:
Thanks!